README.md

```diff
@@ -86,4 +105,4 @@ Fragment that contains several links and information for the user. Links to Your
 
 ### error.html
 
-Generic error page that gives the user an option to email Companies House. For this to work there must be a ```baseLayout.html``` file within the ```layouts/```` directory.
\ No newline at end of file
+Generic error page that gives the user an option to email Companies House. For this to work there must be a ```baseLayout.html``` file within the ```layouts/```` directory.
```

Comment:
What's changed here?

Reply:
New line I am guessing

Reply:
I think my editor adds them into end of file automatically

Reply:
Naughty editor.